### ISSUE TITLE
#680665742------------- All page content should be contained by landmarks

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#1839]
<strong>PageName:</strong> FeedDetailsPage  

<strong>Source:</strong>`<calcite-card thumbnail-position="block-start">`

<strong>Element Location: </strong>calcite-card:nth-child(4)

<strong>Impact: </strong>moderate

<strong>Failure Summary:</strong> Fix any of the following:
  Some page content is not contained by landmarks

<strong>Tags:</strong> cat.keyboard,best-practice

<strong>HelpUrl:<strong>https://dequeuniversity.com/rules/axe/4.6/region?application=axeAPI
<strong>Screenshot:<strong>`C:\Automation\Workspace\Velocity\Screenshot\FeedDetailsPage\680665742_All page content should be contained by landmarks.png`
